tests/calculus-endpoints/social.test.ts

```diff
@@ -0,0 +1,50 @@
+import request from 'supertest'
+import app from '../../src/app'
+import { Credibility, TwitterUser } from '../../src/calculator/models'
+
+interface socialCredibilityParams extends TwitterUser {
+  followersCount: number,
+  friendsCount: number
+}
+
+describe('/calculate/social/scraped endpoint', () => {
+  describe('http 200 calls', () => {
+    function testCredibilityWithOkData(
+      expectedReturn : Credibility, params: socialCredibilityParams) {
+      return request(app)
+        .get('/calculate/social/scraped')
+        .query(params)
+        .expect(200)
+        .expect(expectedReturn)
+    }
+    // describe('zero followers, zero following', () => {
+    //   const params = {
+    //     name: '',
+    //     verified: false,
+    //     yearJoined: 2000      
+    //   }
+    //   it('returns credibility=null', () => {
+    //     return testCredibilityWithOkData({ credibility: null }, {
+    //       followersCount: 0,
+    //       friendsCount: 0,
+    //       ...params
+    //     })
+    //   })
+    // })
+
+    describe('1.200.000 followers, 421 following', () => {
+      const params = {
+        name: '',
+        verified: false,
+        yearJoined: 2000  
+      }
+      it('returns credibility=30.04999998245834 with', () => {
+        return testCredibilityWithOkData({ credibility: 30.04999998245834 }, {
+          followersCount: 1200000,
+          friendsCount: 421,
+          ...params
+        })
+      })
+    })
```

Comment:
Mmm I see what are you trying to do in this tests, but I encourage you to use more handable numbers that results in easier credibilities to test with.

For example, you could try this simple combinations:
```
followers = ANY VALUE YOU WANT
following = 0

followers = 0
following = ANY VALUE YOU WANT

followers = ANY VALUE YOU WANT
following = ANY VALUE YOU WANT
```
The ANY VALUE YOU WANT should be values whose final calculations returns a more handable number, not `30.04999998245834`. The test could fail also for floating-point precission and that is sad.

Reply:
Thanks for the info! Also found out that my error is also related to the way I send the parameters, my route is expecting two and I am only sending one!